cartography/intel/aws/ecs.py

```diff
@@ -0,0 +1,590 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+import boto3
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.util import camel_to_snake
+from cartography.util import dict_date_to_epoch
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_ecs_clusters(boto3_session: boto3.session.Session, region: str) -> Tuple[List, List]:
+    client = boto3_session.client('ecs', region_name=region)
+    paginator = client.get_paginator('list_clusters')
+    clusters: List[Any] = []
+    cluster_arns: List[Any] = []
+    for page in paginator.paginate():
+        cluster_arns.extend(page.get('clusterArns', []))
+    # TODO: also include attachment info, and make relationships between the attachements
+    # and the cluster.
+    includes = ['SETTINGS', 'CONFIGURATIONS']
+    for i in range(0, len(cluster_arns), 100):
```

Comment:
What's the chunking for? Throttling?

Reply:
The API takes a maximum of 100 arns in the argument